doc/en/user/source/introduction/gettinginvolved.rst

```diff
@@ -1,52 +1,66 @@
 .. _getting_involved: 
 
-Getting Involved
+Getting involved
 ================
 
-There are many ways that one can help out with the GeoServer project. GeoServer fully embraces an open source development model that does not see a split between user and developer, producer and consumer, but instead sees everyone as a valuable resource in a collaborative quest to build something better than any of us could alone. 
+GeoServer exists because of the efforts of people like you.
+
+There are many ways that you can help out with the GeoServer project. GeoServer fully embraces an open source development model that does not see a split between user and developer, producer and consumer, but instead sees everyone as a valuable contributor.
 
 Development
 -----------
 
-Helping to develop GeoServer is the obvious way to help out. Developers usually start with bug fixes and small patches, and then move into larger contributions as they learn the system. Our developers are more than happy to help out  - try the Developers mailing list below - as you learn and get acquainted. We try our hardest to keep our code clean and well documented. You can find the project on `github <https://www.github.com/geoserver/>`_.
+Helping to develop GeoServer is the obvious way to help out. Developers usually start with bug fixes and other small patches, and then move into larger contributions as they learn the system. Our developers are more than happy to help out as you learn and get acquainted. We try our hardest to keep our code clean and well documented.
+
+You can find the project on `GitHub <https://www.github.com/geoserver/>`_. As part of the GitHub model, anyone can submit patches as pull requests, which will be evaluated by the team. To learn more about contributing to the GeoServer codebase, we highly recommend joining the GeoServer developers mailing list. See details below.
 
 Documentation
 -------------
 
-One of the best and most needed ways to help out is with the documentation. Our official documentation is contained as part of our official code repository in order to maintain a uniform look and feel.
+Another crucial way to help out is with documentation. Whether it's adding tutorials or just correcting mistakes, every contribution serves to make the project more healthy. And the best part is that you do not need to be a developer in order to contribute.
+
```

Comment:
It may be worth linking to http://docs.geoserver.org/latest/en/docguide/quickfix.html here.